src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/AuthorMap/AuthorMap.xaml.cs

```diff
@@ -0,0 +1,497 @@
+                    MessageBox.Show("Unable to save map updates: " + ex.Message);
+                }
+                finally
+                {
+                    // Hide the progress bar
+                    SaveProgressBar.Visibility = Visibility.Hidden;
+                }
+            }
+        }
+
+        private void ClearMap(object sender, RoutedEventArgs e)
+        {
+            // Set the map to null
+            _myMap = null;
+
+            // Show a plain gray map in the map view
+            MyMapView.Map = new Map(Basemap.CreateLightGrayCanvas());
+        }
+        
+        private void UpdateViewExtentLabels()
+        {
+            // Get the current view point for the map view
+            Viewpoint currentViewpoint = MyMapView.GetCurrentViewpoint(ViewpointType.BoundingGeometry);
+            if (currentViewpoint == null) { return; }
+
+            // Get the current map extent (envelope) from the view point
+            Envelope currentExtent = currentViewpoint.TargetGeometry as Envelope;
+            // Project the current extent to geographic coordinates (longitude / latitude)
+            Envelope currentGeoExtent = GeometryEngine.Project(currentExtent, SpatialReferences.Wgs84) as Envelope;
+
+            // Fill the app text boxes with min / max longitude (x) and latitude (y) to four decimal places
+            XMinTextBox.Text = currentGeoExtent.XMin.ToString("0.####");
+            YMinTextBox.Text = currentGeoExtent.YMin.ToString("0.####");
+            XMaxTextBox.Text = currentGeoExtent.XMax.ToString("0.####");
+            YMaxTextBox.Text = currentGeoExtent.YMax.ToString("0.####");
+        }
+
+        #region OAuth helpers
+        private void UpdateAuthenticationManager()
+        {
+            // Register the server information with the AuthenticationManager
+            Esri.ArcGISRuntime.Security.ServerInfo portalServerInfo = new ServerInfo
+            {
+                ServerUri = new Uri(ServerUrl),
+                OAuthClientInfo = new OAuthClientInfo
+                {
+                    ClientId = AppClientId,
+                    RedirectUri = new Uri(OAuthRedirectUrl)
+                },
+                // Specify OAuthAuthorizationCode if you need a refresh token (and have specified a valid client secret)
+                // Otherwise, use OAuthImplicit
+                TokenAuthenticationType = TokenAuthenticationType.OAuthImplicit
+            };
+            AuthenticationManager.Current.RegisterServer(portalServerInfo);
+
+            // Use the OAuthAuthorize class in this project to create a new web view that contains the OAuth challenge handler.
+            AuthenticationManager.Current.OAuthAuthorizeHandler = new OAuthAuthorize();
+
+            // Create a new ChallengeHandler that uses a method in this class to challenge for credentials
+            AuthenticationManager.Current.ChallengeHandler = new Esri.ArcGISRuntime.Security.ChallengeHandler(CreateCredentialAsync);
```

Comment:
Is it necessary to fully qualify this statement with 'Esri.ArcGISRuntime.Security' or will the shorthand version work:
 Ex: AuthenticationManager.Current.ChallengeHandler = new ChallengeHandler(CreateCredentialAsync);